dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/BitstreamRestRepository.java

```diff
@@ -248,4 +255,25 @@ public BundleRest performBitstreamMove(Context context, Bitstream bitstream, Bun
 
         return converter.toRest(targetBundle, utils.obtainProjection());
     }
+
+    /**
+     * Method that will transform the provided PATCH json body into a list of operations.
+     * The operations will be handled by a supporting class resolved by the
+     * {@link org.dspace.app.rest.repository.patch.ResourcePatch#patch} method.
+     *
+     * @param context The context
+     * @param jsonNode the json body provided from the request body
+     */
+    public void patchBitstreamsInBulk(Context context, JsonNode jsonNode) throws SQLException {
+        int operationsLimit = configurationService.getIntProperty("patch.operations.limit", 1000);
```

Comment:
This appears to be a new configuration, but it's not added to any of the *.cfg files?  It should be added to our configs and documented there.

We may want to rename this `rest.patch.operations.limit` and add it to the `rest.cfg` file (in the top section for the "RESTv7", perhaps just before all the `rest.properties.exposed` values).